.github/templates/model_regression_test_config_comment.tmpl

```diff
@@ -9,6 +9,12 @@ Comment with a help message is added if a PR doesn't contain a comment
 with a configuration for the model regression tests.
 
  */ -}}
+{{ define "check_available_configuration" -}}
+NLU
+{{- if has .dataset "domain" -}}
```

Comment:
@kedz Currently, I'm checking if the mapping file includes the `domain` field for a given dataset in order to determine if the dataset includes `core` or not

Is it correct? Or there is a better way to determine if a dataset includes `core` or not?

Reply:
I think that should work!